doc/connection_and_messaging_reliability.md

```diff
@@ -288,23 +288,68 @@ To access it the user can invoke one of the functions below, passing a callback
 
 On [iothub\_client](https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client.h#L171) module:
 
+```c
 IOTHUB\_CLIENT\_RESULT IoTHubClient\_SetConnectionStatusCallback(IOTHUB\_CLIENT\_HANDLE iotHubClientHandle, IOTHUB\_CLIENT\_CONNECTION\_STATUS\_CALLBACK connectionStatusCallback, void\* userContextCallback);
+```
 
 On [iothub\_client\_ll](https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client_ll.h#L402) module:
 
+```c
 IOTHUB\_CLIENT\_RESULT IoTHubClient\_LL\_SetConnectionStatusCallback(IOTHUB\_CLIENT\_LL\_HANDLE iotHubClientHandle, IOTHUB\_CLIENT\_CONNECTION\_STATUS\_CALLBACK connectionStatusCallback, void\* userContextCallback);
+```
 
 This callback will be invoked in these specific situations:
 
 - When the device client gets connected or re-connected to the Azure IoT Hub;
 
 - When the device client gets disconnected due to any issues;
 
-> These include network availability and IoT hub connectivity issues, authentication failures.
+  > These include network availability and IoT hub connectivity issues, authentication failures.
 
 - When the device client ceases attempting to re-connect (if the Retry Policy no longer allows to).
 
-Please take a look [here](https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client_ll.h#L144) for more info on the possible status and reasons.
+The type `IOTHUB\_CLIENT\_CONNECTION\_STATUS\_CALLBACK` is [defined](https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client_ll.h#L183) as:
```

Comment:
```suggestion
The type `IOTHUB_CLIENT_CONNECTION_STATUS_CALLBACK` is [defined](https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client_ll.h#L183) as:
```

Reply:
I have similar feedback below -- the link above to `https://github.com/Azure/azure-iot-sdk-c/blob/2018-05-04/iothub_client/inc/iothub_client_ll.h#L183` - this won't change, but pointing to very old copy of SDK.

I think its best to just point to the header file but not line #'s.  If someone needs more than what docs have, they can easily search header for it.

Reply:
Oh, that's too bad. It's so neat to point to the exact line in the code when we reference it. I agree we need to get rid of the references to old releases (branch/tag 2018-05-04), but I will keep the line references. I don't expect them to shift that much  anyway, because we don't have much code churn in the C SDK anymore..

Reply:
Removed all references to the old branch, and pointed to `main`.